src/main/java/uk/gov/justice/probation/courtcasematcher/service/MatcherService.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.justice.probation.courtcasematcher.service;
 
+import com.google.common.eventbus.EventBus;
```

Comment:
This is the only change in this file so I think it must be unused

Reply:
Will do. TBH - thought IntelliiJ sorted that out for me pre-commit.